core/src/main/java/org/elasticsearch/search/suggest/term/TermSuggestionBuilder.java

```diff
@@ -65,12 +72,20 @@ public TermSuggestionBuilder(String name) {
      * tokens in the suggest text.
      * </ol>
      */
-    public TermSuggestionBuilder suggestMode(String suggestMode) {
+    public TermSuggestionBuilder suggestMode(@Nullable SuggestMode suggestMode) {
```

Comment:
Wherever makes the most sense really. In this case I would put the default constants in `DirectSpellcheckerSettings` I think